src/main/webapp/i18n/de/global.json

```diff
@@ -178,6 +178,7 @@
             "back": "Zurück",
             "cancel": "Abbrechen",
             "delete": "Löschen",
+            "noButtonTextDelete": "",
```

Comment:
Since I can not change the formatting of it text on the jhiDeleteButton, I need to add the button text manually. To make "delete" not appear twice I need to set it to an empty string here.